runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomPlugin.java

```diff
@@ -51,6 +63,18 @@
 public class KingdomPlugin extends Plugin
 {
 	private static final ImmutableSet<Integer> KINGDOM_REGION = ImmutableSet.of(10044, 10300);
+	private static final String CONFIG_GROUP_NAME = "kingdomofmiscellania";
```

Comment:
Define this as a constant in the config interface, and reference it both in the `@ConfigGroup` definition and here in the plugin.